editoast/src/infra_cache/operation/update.rs

```diff
@@ -187,7 +193,12 @@ mod tests {
     async fn invalid_update_track() {
         let db_pool = DbConnectionPoolV2::for_tests();
         let infra = create_empty_infra(db_pool.get_ok().deref_mut()).await;
-        let track = create_track(db_pool.get_ok().deref_mut(), infra.id, Default::default()).await;
+        let track = create_infra_object::<TrackSection>(
+            db_pool.get_ok().deref_mut(),
+            infra.id,
+            Default::default(),
+        )
```

Comment:
You should be able to do this instead (and for all the other occurences)?

```suggestion
        let track = create_infra_object(
            db_pool.get_ok().deref_mut(),
            infra.id,
            TrackSection::default(),
        )
```

Reply:
Done.